### PR TITLE
Adds sortBy and order query params to GET /cases

### PR DESCRIPTION
**Pull request by murilovmachado**
## Description
This PR adds two query params to GET /cases:
- `sortBy`: the column to be sorted
- `order`: ASC or DESC

### Checklist
- [x] Corresponding issue has been opened
- [x] New tests added

### Related Issues
Backend part for https://bugs.benetech.org/browse/CHI-1116

### Verification steps
This is easier tested when the front-end PR is implemented, but if you want to try, follow the steps below:
- Run hrm locally
- Run front-end pointing to local hrm
- Modify front-end to play with the query params: `sortBy` and `order`

### CODE REVIEW

**Review by stephenhand — CHANGES_REQUESTED** [review]
LGTM but will need changes once #155 is merged

File: controllers/helpers.js
Comment (stephenhand) [inline controllers/helpers.js:37]: Could we make it so it always sorts by `createdAt DESC` as a default, and as a secondary sort when a different one is specified?

So if you specify no sort, the sort will be

 `ORDER BY "Cases".createdAt DESC`
 
 But if you specify to sort by status, the order will be
 
 `ORDER BY "Cases".status ASC, "Cases".createdAt DESC`
 
 So all the open cases aren't jumbled up in a random order etc
 
 On second thoughts maybe it's better to stick with `ìd` rather than `createdAt` for the time being, since `createdAt` is not indexed. I want to move to `createdAt` in case we move away from an ascending integer as an ID, but that can wait until we review the indexes.

**Comment by stephenhand** [comment]
Perhaps we could add a service test exercising the sort, in addition to the controller unit tests?

File: controllers/helpers.js
Comment (murilovmachado) [reply controllers/helpers.js:37]: I made `sortBy` defaults to `id` because that was specified in the Jira ticket. Right now, the result is the same as using `createdAt`, because the ids are sequential. But, if we change ids to not be sequential anymore, than it may get a little bit confusing.

Comment (stephenhand) [reply controllers/helpers.js:37]: Yes Id is fine for now, it's keeping it as a secondary sort clause when you are sorting on something else is the important thing

**Pull request merged**
